src/rda/map.ts

```diff
@@ -29,16 +32,29 @@ export interface MuRDAMapTypes<
         id:KeySchema;
         action:ValueRDA['actionSchema'];
     }>;
+    moveActionSchema:MuStruct<{
+        src:KeySchema;
+        dst:KeySchema;
+    }>;
+    unmoveActionSchema:MuStruct<{
+        overwritten:MuBoolean;
+        dst:KeySchema;
```

Comment:
The id should always be a string.  You don't need to store the key in a schema in any action.

Actions should only reference entries using their unique id.